src/main/java/cuchaz/enigma/translation/mapping/EntryMapping.java

```diff
@@ -4,29 +4,29 @@
 import javax.annotation.Nullable;
 
 public class EntryMapping {
-	private final String targetName;
+	private final @Nullable String targetName;
```

Comment:
I know its nothing to do with this PR, but we should move to checker for these annotations at somepoint.